OpenRA.Mods.Common/Traits/Mobile.cs

```diff
@@ -30,10 +30,10 @@ public class MobileInfo : PausableConditionalTraitInfo, IMoveInfo, IPositionable
 		[Desc("Which Locomotor does this trait use. Must be defined on the World actor.")]
 		public readonly string Locomotor = null;
 
-		public readonly int InitialFacing = 0;
+		public readonly WAngle InitialFacing = WAngle.Zero;
 
 		[Desc("Speed at which the actor turns.")]
-		public readonly int TurnSpeed = 255;
+		public readonly WAngle TurnSpeed = new WAngle(1023);
```

Comment:
Anything higher than 512 is pointless, so maybe we should change the default value here?